.circleci/config.yml

```diff
@@ -396,14 +395,9 @@ commands:
 
 jobs:
   spm-release-build-xcode-14:
-    <<: *base-job
-    parameters:
-      xcode_version:
-        type: string
-        default: '14.3.0'
-      install_swiftlint:
-        type: boolean
-        default: false
+    executor:
+      name: macos-executor
+      xcode_version: '14.3.0'
```

Comment:
regular usage is just so much cleaner

Reply:
Yea, I think it's not a bad idea to use aliases only sparingly, because of the inflexibility they often introduce.